src/DotLiquid.Tests/Tags/StandardTagTests.cs

```diff
@@ -81,6 +81,36 @@ public void TestForWithNestedDictionary()
                 }
             } };
 
+            Helper.AssertTemplateResult("JaneMike", "{% for item in People %}{{ item.Value.First }}{%endfor%}",

```

Comment:
Formatting should be cleaned-up.
If we move parameters to multiple lines, each parameter should start on a new line.

Reply:
... and use named parameters as we have at least two strings.